net.rpcs3.RPCS3.yaml

```diff
@@ -96,9 +96,12 @@ modules:
     sources:
       - type: git
         url: https://github.com/RPCS3/llvm-mirror.git
-        commit: c725f494c91611018f5d830eca22c0a1662c0f31
+        branch: master
+        commit: 9b52b6c39ae9f0759fbce7dd0db4b3290d6ebc56
```

Comment:
The branch should not be specified for rpcs3-llvm. Sometimes, they wait for days and even weeks to update it in rpcs3's repo. If the commit is newer than the one expected by rpcs3, the build will fail.

Reply:
The linter require one unfortunately...
If we define just `master`, won't whatever latest commit in `master` will be fetched anyway?

Reply:
Doesn't solve the problem.

Ideally, we should extract the expected commit from the submodules list of rpcs3 git repo. But I don't know if that's possible with fedc.

Reply:
Oh ok I see, the commit has be to the one defined in the rpcs3 repo, or one older.
It is assumed that there is some sort of backward compatibility then?
has it been tried to set `-DBUILD_LLVM_SUBMODULE=ON` in the `rpcs3` module instead?

Reply:
That's another possibility. But rpcs3-llvm is only a build-time dependency. We have to make sure to strip it from the final export.

Reply:
Should be doable with the `cleanup` field.
I'll look into that.

Reply:
Thanks for taking care of this.

Reply:
I got it working. I tried without the additional `cflags`, but it was crashing on a game.
I didn't have the issue after adding them back.

Reply:
I don't see the cleanup section 🤔

Reply:
I checked the built files and it wasn't necessary, I haven't seen anything in /bin, /lib or /share.
I suppose the built custom LLVM is kept inside the build directory, not installed system-wide.